dlrover/python/master/hyperparams/simple_strategy_generator.py

```diff
@@ -115,18 +118,35 @@ def _generate_dataloader_config(
                 updated_batch_size = batch_size
 
             logger.info(f"updated_batch_size: {updated_batch_size}")
-            return DataLoaderConfig(
+            update_dataloader_config = DataLoaderConfig(
                 updated_version,
                 dataloader_config.dataloader_name,
                 updated_batch_size,
                 0,
                 0,
             )
-        else:
-            return dataloader_config
 
-    def _generate_optimizer_config(self):
-        return OptimizerConfig(5, 6)
+            try:
+                ratio = updated_batch_size / batch_size
+            except ZeroDivisionError:
+                ratio = 1
+            coefficient = math.sqrt(ratio)
+
+            update_version = optimizer_config.version + 1
+            update_learning_rate = optimizer_config.learning_rate * coefficient
+            update_weight_decay = optimizer_config.weight_decay * coefficient
+
+            logger.info(f"update_learning_rate: {update_learning_rate}")
+            logger.info(f"update_weight_decay: {update_weight_decay}")
```

Comment:
`logger.info(f"Update optimizer with learning rate {update_learning_rate} and weight decay {update_weight_decay}")`